_includes/pbc/all/install-features/latest/install-the-ssp-asset-management-feature.md

```diff
@@ -380,23 +563,149 @@ console data:import glossary
+AST--3,OfficeJet Pro 9025e All-in-One Printer,CN1234ABCD,"The OfficeJet Pro 9025e is a high-performance multifunctional printer designed for modern office environments. It offers fast printing, scanning, copying, and faxing capabilities with automatic duplex printing. With built-in Wi-Fi and mobile printing options, this all-in-one device enhances workplace efficiency.",https://d2s0ynfc62ej12.cloudfront.net/image/AdobeStock_125577546.jpeg,spryker_systems_HR,spryker_systems_HR
+AST--4,Logistic Casa F-08,,1FUJGLDR5KL123456,https://d2s0ynfc62ej12.cloudfront.net/image/AdobeStock_223498915.jpeg,spryker_systems_HR,spryker_systems_HR
+```
+
+| COLUMN                      | REQUIRED | DATA TYPE | DATA EXAMPLE                                                  | DATA EXPLANATION                                                          |
+|-----------------------------|----------|-----------|---------------------------------------------------------------|---------------------------------------------------------------------------|
+| reference                   | ✓        | string    | AST--1                                                        | Unique identifier for the asset used as a reference in the system.        |
+| name                        | ✓        | string    | DemoBrand Print Pro 2100                                      | The display name of the asset.                                            |
+| serial_number               | x        | string    | PRNT000014                                                    | The serial number of the asset for identification purposes.               |
+| note                        | x        | string    | The DemoBrand Print Pro 2100...                               | Detailed description or notes about the asset.                            |
+| external_image_url          | x        | string    | https://d2s0ynfc62ej12.cloudfront.net/image/Demo_Printer.jpeg | URL to an external image of the asset.                                    |
+| business_unit_key           | ✓        | string    | spryker_systems_HR                                            | The key of the business unit that owns the asset.                         |
+| assigned_business_unit_keys | x        | string    | spryker_systems_HR                                            | Comma-separated list of business unit keys that have access to the asset. |
```

Comment:
Is it really so that business_unit_key is required and assigned_business_unit_keys is not? According to business requirement business_unit_key must be in a list of assigned_business_unit_keys. Looks like we have a bug in asset data import

Reply:
The business_unit_key is not required, have fixed that.
There is no validation for the case provided.